lib/liquid/standardfilters.rb

```diff
@@ -377,6 +379,43 @@ def to_date(obj)
       nil
     end
 
+    def nested_respond_to?(input, property)
+      return true if property.empty?
+      p = property.split('.').first
+      q = property[/#{p}.?([\w.]*)/, 1]
+      if input.respond_to?(:[]) && accepts?(input, p)
+        return nested_respond_to?(input[p], q)
+      elsif input.respond_to?(p)
+        return nested_respond_to?(input.send(p), q)
+      end
+      false
+    end
+
+    def nested_send(input, property)
+      return input if property.empty?
+      p = property.split('.').first
+      q = property[/#{p}.?([\w.]*)/, 1]
+      if input.respond_to?(:[]) && accepts?(input, p)
+        return nested_send(input[p], q)
+      elsif input.respond_to?(p)
+        return nested_send(input.send(p), q)
+      end
+      nil
+    end
+
+    def accepts?(input, index)
+      !input[index].nil?
+      rescue TypeError
```

Comment:
`nested_send` is called by the `nested_sort`, which should now be called by `sort` when `property.include? '.'` (line 119, standardfilters.rb) . If you remove it, the tests I wrote will not pass.